slave-base/contrib/bin/run-jnlp-client

```diff
@@ -31,7 +31,7 @@ elif [ "${OPENSHIFT_JENKINS_JVM_ARCH}" == "x86_64"  ]; then
 else
     echo "OPENSHIFT_JENKINS_JVM_ARCH is set to ${OPENSHIFT_JENKINS_JVM_ARCH} so using 32 bit Java"
     alternatives --set java $JVMPath32bit
-    export MALLOC_ARENA_MAX=1
+    export MALLOC_ARENA_MAX=${MALLOC_ARENA_MAX:-1}
```

Comment:
as an fyi, via commit 8f60691a47512bce66d328d0f8ca72612d3bc001 this stemmed from purely duplicating advice from the JBOSS side of the house.

Reply:
I can definitely see that it could make a difference to export MALLOC_ARENA_MAX; I'm just trying to ensure that things are reasonably overridable.